dev/com.ibm.ws.componenttest/autoFVT-defaults/src/ant/launch.xml

```diff
@@ -102,8 +102,8 @@
 		<!-- In Java 16, the default enforcement of strong encapsulation (part of JPMS, which was introduced in Java 9) has changed from 
 		     illegal-access=permit to illegal-access=deny.  This causes many of our FATs that use reflection to fail.  
 		     This change sets up a gated property (illegal.access.permit) so that any builds that take place on Java 15+ use 
-		     illegal-access=permit, but older versions will be skipped -->
-		<condition property="illegal.access.permit" value="--illegal-access=permit">
+		     illegal-access=permit, but older versions will be skipped --> 
```

Comment:
The comment should be updated becuase it is clearly wrong now with this change.

Reply:
Good catch!  I have cleaned up these comments.